### PR TITLE
Updating containerd on Windows install steps to add Defender Exclusion

### PR DESCRIPTION
**Pull request by marosset**
Updating installation steps for installing containerd on Windows machines.
Windows Defender has been shown to have a very negative performance impact on Windows so we're providing steps on how to exclude Defender from scanning containerd. 
The Docker installation process on Windows adds Defender exclusions so this is brining parity to containerd.

### CODE REVIEW

File: content/en/docs/setup/production-environment/container-runtimes.md [hunk 1]
```diff
@@ -219,30 +219,35 @@ sudo systemctl restart containerd
...
 {{% tab name="Windows (PowerShell)" %}}
+
 ```powershell
 # (Install containerd)
-# download containerd
-cmd /c curl -OL https://github.com/containerd/containerd/releases/download/v1.4.1/containerd-1.4.1-windows-amd64.tar.gz
-cmd /c tar xvf .\containerd-1.4.1-windows-amd64.tar.gz
+# Download containerd
+curl.exe -L https://github.com/containerd/containerd/releases/download/v1.4.1/containerd-1.4.1-windows-amd64.tar.gz -o containerd-windows-amd64.tar.gz
+tar.exe xvf .\containerd-windows-amd64.tar.gz
```
Comment (perithompson) [inline content/en/docs/setup/production-environment/container-runtimes.md:227]: Probably a nit but is it worth changing it not be explicit on the version? 
```suggestion
curl.exe -L https://github.com/containerd/containerd/releases/download/v{version}/containerd-{version}-windows-amd64.tar.gz -o containerd-windows-amd64.tar.gz
tar.exe xvf .\containerd-windows-amd64.tar.gz
```

But other than that lgtm

Comment (perithompson) [reply content/en/docs/setup/production-environment/container-runtimes.md:227]: happy to be wrong on this! 😄

Comment (jsturtevant) [reply content/en/docs/setup/production-environment/container-runtimes.md:227]: the docs do something similar to this for cri-o: https://kubernetes.io/docs/setup/production-environment/container-runtimes/#cri-o

Comment (marosset) [reply content/en/docs/setup/production-environment/container-runtimes.md:227]: I looked at setting this to the 'latest' release but that currently points to contaienrd v1.3.9 which won't work for windows here :(

Comment (marosset) [reply content/en/docs/setup/production-environment/container-runtimes.md:227]: I did think of doing that tho!

Comment (marosset) [reply content/en/docs/setup/production-environment/container-runtimes.md:227]: I can use {version} and add a comment that version should be >= v1.4

Comment (marosset) [reply content/en/docs/setup/production-environment/container-runtimes.md]: updating - PTAL

Comment (marosset) [reply content/en/docs/setup/production-environment/container-runtimes.md]: https://deploy-preview-26600--kubernetes-io-master-staging.netlify.app/docs/setup/production-environment/container-runtimes/#containerd

**Comment by perithompson** [comment]
lgtm too!

File: content/en/docs/setup/production-environment/container-runtimes.md (hunk 1 above)
Comment (perithompson) [reply content/en/docs/setup/production-environment/container-runtimes.md]: Perfect!

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-26600--kubernetes-io-master-staging.netlify.app/docs/setup/production-environment/container-runtimes/#containerd

**Pull request merged**